msi/GitProduct.wxs

```diff
@@ -266,6 +266,8 @@
   <Fragment>
     <SetProperty Id="ExecConfigureCoreAutoCrlf" Value='"[#GitExe]" config -f "[#GfwGitConfig]" core.autocrlf [LINEENDINGS]' Sequence="execute" Before="ExecConfigureCoreAutoCrlf" />
     <CustomAction Id="ExecConfigureCoreAutoCrlf" BinaryKey="WixCA" DllEntry="WixQuietExec$(var.SixtyFourBit)" Execute="deferred" Return="ignore" Impersonate="no" />
+    <SetProperty Id="ExecConfigureSslCAInfo" Value='"[#BashExe]" -c "git config -f /c/ProgramData/Git/config http.sslCAinfo \"/[#ca_bundle.crt]\""' Sequence="execute" Before="ExecConfigureSslCAInfo" />
```

Comment:
> I should be able to use the $ProgramData environment variable

Yes, that would be good. Please note, though, that 1. it is not available on XP (and for the time being, we do support XP, at least until Cygwin drops it), and 2. it uses backslashes, so it *might* need to be converted to forward slashes before `git config` can use it.